usrp_drivers/usrp_driver.cpp

```diff
@@ -260,6 +260,7 @@ void transmit(zmq::context_t &driver_c, USRP &usrp_d, const DriverOptions &drive
     auto time_now = box_time;
     auto sequence_start_time = time_now + delay;
 
+    auto seqn_sampling_time = num_recv_samples/rx_rate;
```

Comment:
I don't think this is the same since seqtime != total 'scope sync time' . Seqtime was, according to docs
> the amount of time for the whole sequence to transmit, until the logic signal switches low on the last pulse in the sequence.

@LiamAG may want to look at your changes here to make sure this still works as intended for taking the AGC signal.

Reply:
Thats a good point. Maybe this field needs a better name

Reply:
or just documented somewhere closer to this point where it's being used. Maybe a good idea to document all parameters in protobuf packets somewhere. Hard to describe them all in inline comments in the proto file.